pom.xml

```diff
@@ -43,6 +43,7 @@
     <version.org.jboss.msc.jboss-msc>1.2.6.Final</version.org.jboss.msc.jboss-msc>
     <version.jboss-logmanager-ext>1.0.0.Alpha3</version.jboss-logmanager-ext>
 
+
```

Comment:
This shouldn't be here, but since I am travelling this week, maybe we can just it ignore it?